src/components/SettingsTreasuryActivateOsnapButton.vue

```diff
@@ -1,24 +1,58 @@
 <script setup lang="ts">
+const { theme } = useSkin();
+
 defineProps<{
   isOsnapEnabled: boolean;
 }>();
+
+// handling some theming here locally so as not to interfere with the global style.scss file
+const activeStyles = computed(() => {
+  return theme.value === 'light'
+    ? {
+        div: 'bg-[hsla(122,100%,45%,0.13)] text-[hsla(122,100%,21%,1)]',
```

Comment:
Using custom colors with not work for our skins on Snapshot. We don't only support light and dark mode, but also custom skins for custom domains, you can test this by changing the key for a skin in domains.json to `localhost`. Please use only available color variable throughout your whole pr.

Custom colors won't work for our skins on Snapshot. We support not only light and dark mode but also custom skins for custom domains. You can test this by changing a key in `domains.json` to `localhost`. 

Please try to use only the available color variables throughout your entire PR.

Reply:
There is also a bug here with `Activated` button on light mode. 
<img width="781" alt="image" src="https://github.com/snapshot-labs/snapshot/assets/51686767/4eabae7f-49c2-4977-9157-a702cf765d7f">